contrib/statesync.bash

```diff
@@ -2,38 +2,43 @@
 # microtick and bitcanna contributed significantly here.
 set -uxe
 
-# set environment variables
+# Set Golang environment variables.
 export GOPATH=~/go
 export PATH=$PATH:~/go/bin
 
-
-# Install Gaia
+# Install Gaia.
 make install
 
-# MAKE HOME FOLDER AND GET GENESIS
+# Initialize chain.
 gaiad init test 
```

Comment:
```suggestion
gaiad init test --chain-id cosmoshub-4
```

as per @glnro 's suggestion. Would suggest committing this after confirmed from a second party